index.js

```diff
@@ -302,13 +303,13 @@ function session(options) {
 
       if (shouldDestroy(req)) {
         // destroy session
-        debug('destroying');
+        debug('destroying session store');
```

Comment:
This is destroying the session, not the session store

Reply:
@dougwilson - addressed, changed the log message